src/cardboard-vr-display.js

```diff
@@ -233,7 +237,7 @@ CardboardVRDisplay.prototype.onResize_ = function(e) {
       'height: ' + Math.min(screen.height, screen.width) + 'px',
       'border: 0',
       'margin: 0',
-      'padding: 0',
+      'padding: 0 10px 10px 0',
```

Comment:
See lines 228 to 231
On Jun 10, 2016 7:27 PM, "chris van wiemeersch" <notifications@github.com>
wrote:

> In src/cardboard-vr-display.js
> <https://github.com/borismus/webvr-polyfill/pull/131#discussion_r66694017>
> :
>
> > @@ -233,7 +237,7 @@ CardboardVRDisplay.prototype.onResize_ = function(e) {
> >        'height: ' + Math.min(screen.height, screen.width) + 'px',
> >        'border: 0',
> >        'margin: 0',
> > -      'padding: 0',
> > +      'padding: 0 10px 10px 0',
>
> should probably add a comment?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/borismus/webvr-polyfill/pull/131/files/6d414da4f66df7f062790f8f79c028b28ac148ad#r66694017>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAeDAz5Z9OGxMeCAxfGgmjQF5eShdq9Dks5qKfLJgaJpZM4IzErh>
> .
>